docs/services/matrix-signal-bridge.md

```diff
@@ -0,0 +1,78 @@
+## Matrix Signal Bridge
+(Only tested on ARM)
+Docker-compose
+```yaml
+version: "3.9"
+
+services:
+  mautrix-signal:
+    container_name: mautrix-signal
+    image: heywoodlh/mautrix-signal
+    restart: always
+    volumes:
+    - ./bridge:/data
+    - ./signald:/signald
+    depends_on:
+      - signald
+    ports:
+      - 29328:29328
+    networks:
+        - matrix
+
+  signald:
+    container_name: signald
+    #image: docker.io/finn/signald
+    image: mik/signald # my self builed image for arm
+    restart: unless-stopped
+    volumes: 
```

Comment:
Please orientate to the other serivces of the AdminGuide use ```"/srv/main/bride/:/data"``` and  ```"/srv/main/bridge/signald:/data"```. This scheme occurs for all volumes